renku_notebooks/api/amalthea_patches/init_containers.py

```diff
@@ -31,14 +31,9 @@ def git_clone(server: "UserServer"):
         {"name": "GIT_CLONE_COMMIT_SHA", "value": server.commit_sha},
         {"name": "GIT_CLONE_BRANCH", "value": server.branch},
         {
-            # used only for naming autosave branch
             "name": "GIT_CLONE_USER__USERNAME",
             "value": server._user.username,
```

Comment:
We can remove this since it is only used to name the autosave branch... I think the comment is still true.